content/index.hbs

```diff
@@ -18,7 +18,7 @@
 					<p>COVID-19 Information</p>
 				</div>
 				<div class="message-body">
-					<p>Due to COVID-19, Detachment 165 and Georgia Tech are implementing certain policy changes and new guidelines. <a href="/covid">More information can be found here.</a></p>

```

Comment:
Not sure what local COVID guidance is, but if we're retiring the COVID info page (https://afrotc.gatech.edu/covid), I think a better way to do things would be to get rid of the alert box and instead update the link for `COVID-19 Info` in the top nav bar to link to the Georgia Tech COVID info page instead.

Reply:
Also, gatech.edu links to http://health.gatech.edu/coronavirus instead, which has more info on campus guidelines so I'd recommend using that